src/proxyd/proxyd_launcher.star

```diff
@@ -58,7 +59,7 @@ def launch(
         observability_helper,
     )
 
-    service = plan.add_service("proxyd-{0}".format(network_params.network), config)
+    service = plan.add_service("proxyd-{0}".format(network_params.network_id), config)
```

Comment:
why is `network_id` the correct one here? i tried that first but switched to `network` for readability

Reply:
"network" is not meant to be unique.
Actually, as far as I can tell its main use is to be a key in that map of exactly 1 element:
https://github.com/ethpandaops/optimism-package/blob/main/src/package_io/constants.star#L28

"network_id" is the piece of data that's unique.

Reply:
I'm not sure that's the case, as there's a `network` field in each L2's `network_params` block. altho I admit I'm a bit confused as to why there's three fields that seem to serve similar purposes:

```yaml
network_params:
  network: "kurtosis"
  network_id: "2151908"
  name: "op-kurtosis"
```

do we need all three, should we have just one mb? where does the `network_id` even come from, should it be generated if do indeed need it?

Reply:
it’s inherited from ethereum package. 
In ethereum package you can also define network as sepolia for example and it will automatically sync sepolia. Thought it could also be done for L2, in case you wanna test something on sepolia for example. 
Network ID is useful also to create unique chain ids. 
name just refers to the prefix of the services.

Reply:
can we merge this fix? without it, we can't deploy multiple L2s right now